pkg/auth/auth.go

```diff
@@ -270,6 +272,13 @@ func authConfigsToAuthFile(authConfigs map[string]types.DockerAuthConfig) (strin
 	return authFilePath, nil
 }
 
+// stripHTTPSPrefix takes a server and removes the leading "http[s]://" prefix.
+// the resulting registry.
+func stripHTTPSPrefix(server string) string {
+	stripped := strings.TrimPrefix(server, "http://")
+	return strings.TrimPrefix(stripped, "https://")
```

Comment:
This turns the proverbial `https://index.docker.io/v1/` into `index.docker.io/v1/`, a repo-namespaced value (and an invalid one at that.)

Reply:
Added normalization cases for docker registries as well.

Reply:
This is still so different from https://github.com/containers/image/blob/4fcb52359b759a276a760fb28c939c357b3aea48/pkg/docker/config/config.go#L732 . Why?

`http{,s}://`-prefixed entries are not namespaced; but it _is_ supposed to be possible to have a `docker.io/vendor` namespaced entry.